pkg/sql/codegen/attribute/attribute.go

```diff
@@ -68,9 +70,27 @@ func (t Type) String() string {
 //   2. Customer checker
 func (d Dictionary) Validate(attrs map[string]interface{}) error {
 	for k, v := range attrs {
+		var desc *Description
 		desc, ok := d[k]
 		if !ok {
-			return fmt.Errorf(errUnsupportedAttribute, k)
+			// Support attribute defination like "model.*" to match attributes start with "model"
+			keyParts := strings.Split(k, ".")
+			if len(keyParts) == 2 {
+				wildCard := fmt.Sprintf("%s.*", keyParts[0])
+				descWild, okWildCard := d[wildCard]
+				if okWildCard {
+					desc = descWild
+				} else {
+					return fmt.Errorf(errUnsupportedAttribute, k)
+				}
+			} else {
+				return fmt.Errorf(errUnsupportedAttribute, k)
+			}
+
+		}
+		// unknown type of attribute do not need to run validate
+		if desc.Type == Unknown {
+			return nil
```

Comment:
why `return`? Should be `continue`?

Reply:
Unknown type do not need to check the actual value's type, just added checker call here.

Reply:
`Validate` should validate all attributes. One attribute being `Unknown` doesn't mean we don't need to check the rest attributes. So we should change `return nil` to `continue`.

Reply:
I see, sorry my brain wa ta le.